### PR TITLE
Add TDVF secure boot key enrolling tool

### PR DESCRIPTION
**Pull request by jialeif**
A python tool to enable secure boot for TDVF/OVMF.
It can enroll PK, KEK, DB certificates and secure boot variable.

Signed-off-by: mxu9 <min.m.xu@intel.com>
Signed-off-by: jialeie <jialei.feng@intel.com>

### CODE REVIEW

File: utils/tdvf-key-enroll/makefile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.DEFAULT_GOAL := wheel
```
Comment (clsulliv) [inline utils/tdvf-key-enroll/makefile:1]: Its pretty standard to have the makefile as Makefile with a capital M.

File: utils/tdvf-key-enroll/tdvfkeyenroll/VarEnroll.py [hunk 2]
```diff
@@ -0,0 +1,1036 @@
...
+      UINT32  Reserved1;
+    } VARIABLE_STORE_HEADER;
+
+    '''
+    _EFI_VARIABLE_GUID = \
+        "ddcf3616-3275-4164-98b6-fe85707ffe7d"
+    _EFI_AUTHENTICATED_VARIABLE_BASED_TIME_GUID = \
+        "aaf32c78-947b-439a-a180-2e144ec37792"
+    _EFI_AUTHENTICATED_VARIABLE_GUID = \
+        "515fa686-b06e-4550-9112-382bf1067bfb"
```
Comment (clsulliv) [inline utils/tdvf-key-enroll/tdvfkeyenroll/VarEnroll.py:356]: Are these GUIDs a standard of some sort? Where do they come from?

Comment (jialeif) [reply utils/tdvf-key-enroll/tdvfkeyenroll/VarEnroll.py:356]: Yes, they've been defined by TDVF. For example 
EFI_VARIABLE_GUID and EFI_AUTHENTICATED_VARIABLE_GUID can be found in
https://github.com/tianocore/edk2-staging/blob/2022-ww05.5/MdeModulePkg/Include/Guid/VariableFormat.h#L13

Comment (jialeif) [reply utils/tdvf-key-enroll/tdvfkeyenroll/VarEnroll.py:356]: Thanks for your review, I've updated

**Review by kenplusplus — APPROVED** [review]
LGTM

**Review by clsulliv — APPROVED** [review]

**Pull request merged**